platform-includes/llm-rules-logs/javascript.node.mdx

```diff
@@ -3,11 +3,10 @@
 
 ````markdown {filename:rules.md}
 
-# Logs 
+# Logs
 
-- Where logs are used, ensure Sentry is imported using `import * as Sentry from "@sentry/node"` 
+- Where logs are used, ensure Sentry is imported using `import * as Sentry from "@sentry/node"`
 - Reference the logger using `const { logger } = Sentry`
-- Sentry initialization needs to be updated to include the `logger` integration.
```

Comment:
Lets still include that `enableLogs: true` should be set within the `_experiments` configuration block to reinforce the snippet below. Calling it a logger integration was wrong in my case.